llvm/include/llvm/CodeGen/GlobalISel/IRTranslator.h

```diff
@@ -624,6 +624,10 @@ class IRTranslator : public MachineFunctionPass {
 
   StackProtectorDescriptor SPDescriptor;
 
+  const TargetLowering &getTargetLowering() {
+    return *MF->getSubtarget().getTargetLowering();
```

Comment:
this can just be a simple member in IRTranslator, no need to query through the subtarget every time